index.html

```diff
@@ -0,0 +1,321 @@
+                including keyword, template, and semantic queries.</li>
+            <li>Be scalable to large databases of TDs.</li>
+            </ul></p>
+        </section>
+        <section id="requirements-data" class="informative">
+            <h2>Data Management</h2>
+            <p><ul>
+            <li>Support both Dynamic and Static TDs.</li>
+            <li>Support explicit deletion and access control management of TDs.</li>
+            <li>Automatically clean up TDs for devices that are no longer accessible.</li>
+            </ul></p>
+        </section>
+        <section id="requirements-security" class="informative">
+            <h2>Security</h2>
+            <p><ul>
+            <li>Support best known methods for confidentiality and authentication.</li>
+            <li>Support authorization and role management.</li>
+            </ul></p>
+        </section>
+        <section id="requirements-privacy" class="informative">
+            <h2>Privacy</h2>
+            <p><ul>
+            <li>Support authentication and authorization mechanisms that do not reveal user identities.</li>
+            <li>Support device and information lifecycle, trust management, access controls.</li>
+            <li>Distribute TDs and other metadata only to authenticated and authorized users.</li>
+            <li>Don’t leak metadata or queries to unauthorized entities.</li>
+            </ul></p>
+        </section>
+        <section id="requirements-alignment" class="informative">
+            <h2>Alignment with Existing Standards</h2>
+            <p><ul>
+            <li>Align with IETF CoRE Resource Directories, CoRE Link Format, and DID.</li>
+            <li>Be accessible via a variety of existing discovery mechanisms,
+                including DNS-SD, DNS-SRV, DHCP, QR codes, and Bluetooth beacons.</li>
+            </ul></p>
+        </section>
+    </section>
+    <section id="architecture" class="informative">
+        <h1>Architecture</h1>
+        <p>Two-Phase approach.
+        </p>
+    </section>
+    <section id="introduction-mech" class="normative">
+        <h1>Introduction Mechanisms</h1>
+        <p>Description of supported introductions, and requirements for 
+        new introduction mechanisms.
+        </p>
+        <section id="introduction-direct" class="normative">
+            <h2>Direct</h2>
+            <p>Any mechanism that results in a single URL.
+               This includes Bluetooth beacons, QR codes, and written URLs to be 
+               typed by a user.
+               A GET on all such URLs MUST result in a TD.
+               For self-describing Things, this can be the TD of the Thing itself.
+               If the URL references a Directory, this MUST be the TD of the
+               Directory service.
+               A Directory can be distinguished from a Thing by the use of an
+               <code>@type</code> including the semantic term <code>WoT-Directory</code>.
```

Comment:
I believe lines 209-214 describe the beginning of the exploration phase and are subject to access control. If that is correct, it will not be possible to distinguish a Directory from a Thing within Direct Introduction without something distinctive inside the URL. I remember discussing well-known paths for this purpose.

Reply:
That is a useful point, but... I can imagine introductions that need authentication, so

Reply:
we can resolve in future PRs.   Needs more discussion